fe2-android/app/src/test/java/com/github/dedis/popstellar/testutils/UITestUtils.java

```diff
@@ -0,0 +1,90 @@
+package com.github.dedis.popstellar.testutils;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertEquals;
+
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.widget.Button;
+
+import androidx.annotation.StringRes;
+import androidx.appcompat.app.AlertDialog;
+import androidx.test.core.app.ApplicationProvider;
+
+import org.robolectric.shadows.ShadowDialog;
+import org.robolectric.shadows.ShadowToast;
+
+/** This class holds utility functions when retrieving particular elements of a view in a test */
+public class UITestUtils {
+
+  /**
+   * Assert that the latest toast showed the provided text
+   *
+   * @param resId resource of the text
+   * @param args arguments to the resource
+   */
+  public static void assertToastIsShown(@StringRes int resId, Object... args) {
+    assertThat("No toast have been displayed", ShadowToast.getLatestToast(), notNullValue());
```

Comment:
The message doesn't seem to align with the function title 🤔 

Also, English fix, if this is actually what you want to say :)
```suggestion
    assertThat("No toast has been displayed", ShadowToast.getLatestToast(), notNullValue());
```
or
```suggestion
    assertThat("No toast was displayed", ShadowToast.getLatestToast(), notNullValue());
```

Reply:
Would `assertToastIsDisplayedWithText()` be better ?